src/backend/distributed/transaction/backend_data.c

```diff
@@ -826,6 +822,21 @@ AssignGlobalPID(void)
 
 	SpinLockAcquire(&MyBackendData->mutex);
 	MyBackendData->globalPID = globalPID;
+	MyBackendData->distributedCommandOriginator = distributedCommandOriginator;
+	SpinLockRelease(&MyBackendData->mutex);
+}
+
+
+/*
+ * OverrideBackendDataWorkerQuery should only be used for isolation testing.
```

Comment:
Can you add a test to set an example?

Reply:
(It might be that isolation tests do not show up in coverage reports, I vaguely remember this)